nautobot/docs/release-notes/version-1.3.md

```diff
@@ -71,12 +75,13 @@ As Python 3.6 has reached end-of-life, the default Docker images published for t
 - [#916](https://github.com/nautobot/nautobot/issues/916) - A Job.Meta.description can now contain markdown-formatted multi-line text.
 - [#1107](https://github.com/nautobot/nautobot/issues/1107) - Circuit Provider account numbers can now be up to 100 characters in length.
 - [#1252](https://github.com/nautobot/nautobot/pull/1252) - As Python 3.6 has reached end-of-life, the default Docker images published for this release (i.e. `1.3.0`, `stable`, `latest`) have been updated to use Python 3.7 instead.
-- [#1268](https://github.com/nautobot/nautobot/issues/1268) - Drop Support for Python 3.6
-- [#1277](https://github.com/nautobot/nautobot/issues/1277) - Upgraded Django dependency to 3.2.X LTS.
+- [#1268](https://github.com/nautobot/nautobot/issues/1268) - Drop Support for Python 3.6.
+- [#1277](https://github.com/nautobot/nautobot/issues/1277) - Updated Django dependency to 3.2.X LTS.
 - [#1307](https://github.com/nautobot/nautobot/pull/1307) - Updated various Python package dependencies to their latest compatible versions.
 - [#1314](https://github.com/nautobot/nautobot/pull/1314) - Updated various development-only Python package dependencies to their latest compatible versions.
 - [#1321](https://github.com/nautobot/nautobot/pull/1321) - Updates to various browser package dependencies. This includes updating from Material Design Icons 5.x to 6.x, which has a potential impact on plugins: a [small number of icons have been removed or renamed](https://dev.materialdesignicons.com/upgrade#5.9.55-to-6.1.95) as a result of this change.
 - [#1367](https://github.com/nautobot/nautobot/pull/1367) - Extracted Job-related models to submodule `nautobot.extras.models.jobs`; refined Job testing best practices.
+- [#1391](https://github.com/nautobot/nautobot/pull/1391) - Updated Jinja2 dependency to 3.0.X.
```

Comment:
Copypasta I usually make too
```suggestion
- [#1391](https://github.com/nautobot/nautobot/issues/1391) - Updated Jinja2 dependency to 3.0.X.
```

Reply:
Because it's already mentioned in the above section?